Lib/zipimport.py

```diff
@@ -268,50 +209,6 @@ def is_package(self, fullname):
         return mi
 
 
-    # Load and return the module named by 'fullname'.
-    def load_module(self, fullname):
```

Comment:
All of your news entries only mention the ``find_*``-related methods, but not ``load_module()``.

Reply:
Oh right, I forgot this one! I updated the PR.